app/models/role.rb

```diff
@@ -141,871 +141,768 @@ def self.permissions_for_access(access)
+        sub_category: 'User Rights',
       },
       can_enable_2fa: {
         description: 'Ability to enable Two-factor authentication for own account',
         administrative: false,
-        categories: [
-          'Administration',
-        ],
+        category: 'Administration',
+        sub_category: 'Account Security',
       },
       enforced_2fa: {
         description: 'Users with this permission will be unable to login until Two-factor authentication has been enabled',
         administrative: false,
-        categories: [
-          'Administration',
-        ],
+        category: 'Administration',
+        sub_category: 'Account Security',
       },
       training_required: {
         description: 'Users with this permission will be unable to login until they have completed user training.',
         administrative: false,
-        categories: [
-          'Administration',
-        ],
+        category: 'Administration',
+        sub_category: 'Account Security',
       },
       can_edit_roles: {
         description: 'Ability to add and remove roles and assign permissions to all roles',
         administrative: true,
-        categories: [
-          'Administration',
-        ],
+        category: 'Administration',
+        sub_category: 'User Rights',
       },
       can_edit_collections: {
         description: 'Ability to add and remove groups and assign entities to all groups',
         administrative: true,
-        categories: [
-          'Administration',
-        ],
+        category: 'Administration',
+        sub_category: 'User Rights',
       },
       can_audit_users: {
         description: 'Access to the audit logs for users',
         administrative: true,
-        categories: [
-          'Administration',
-        ],
+        category: 'Administration',
+        sub_category: 'Auditing',
       },
       can_view_full_ssn: {
         description: 'Ability to see the full Social Security Number for clients',
         administrative: false,
-        categories: [
-          'Client Access',
-        ],
+        category: 'Client Access',
+        sub_category: 'Privacy',
       },
       can_view_full_dob: {
         description: 'Ability to see the Date of Birth for clients',
         administrative: false,
-        categories: [
-          'Client Access',
-        ],
+        category: 'Client Access',
+        sub_category: 'Privacy',
       },
       can_view_hiv_status: {
         description: 'Ability to see the HIV information for clients',
         administrative: false,
-        categories: [
-          'Client Access',
-        ],
+        category: 'Client Access',
+        sub_category: 'Privacy',
       },
       can_view_dmh_status: {
         description: 'Ability to see the Mental Health information for clients',
         administrative: false,
-        categories: [
-          'Client Access',
-        ],
+        category: 'Client Access',
+        sub_category: 'Privacy',
```

Comment:
`Sensitive Client Data`? Idk, "Privacy" feels broad to me. But I think its ok.
I love the sub-categories, so much easier to navigate!